tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php

```diff
@@ -384,7 +387,7 @@ public function addProductsCombinationsToCart(
 
     /**
      * @When I select :countryIsoCode address as delivery and invoice address for customer :customerReference in cart :cartReference
-     * @Given cart :cartReference delivery and invoice address country for customer :customeReferenceis is :countryIsoCode
+     * @Given cart :cartReference delivery and invoice address country for customer :customerReference is :countryIsoCode
```

Comment:
It did not seem to bother, maybe this statement is never used and could be removed? (just the annotation line)

Reply:
Used in `tests/Integration/Behaviour/Features/Scenario/Order/add_cart_rule_to_order.feature`